### PR TITLE
halfway through rating feature

### PR DESCRIPTION
**Pull request by PrettyPearl2302**
## Description
In this pull request, I started to build the rating feature for episodes for my website. I created a rating model in my database, created a backend route to post the ratings, and created the front-end component for the stars (through which the rating happens). When a user clicks on a star, the value of the star gets stored in the database and is recorded as the rating.

 I attempted to not let the user submit more than one rating, by using a useState to determine if they had rated or not, but the issue with my approach is that on page refresh, the state refreshes, which means they can still submit more than one rating. I've been thinking about how to fix that. I'm thinking of doing the check on whether the user has rated or not on the backend instead, just like how I wrote my user sign/login where I make a request to the database to check if that email or username already exists. In this case, I would search the rating table to check if there is a rating value with this user and with this episodeId. 

## Milestones
Building episode rating feature. 

## Resources
https://www.geeksforgeeks.org/how-to-create-a-rating-component-in-reactjs/ (i used this exact format to create the stars)
https://styled-components.com/docs/basics#getting-started

## Test Plan
Screenshot 2023-07-27 at 1 36 14 PM
Screenshot 2023-07-27 at 1 36 46 PM
Screenshot 2023-07-27 at 1 37 31 PM

### CODE REVIEW

File: timify-api/route/comments.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ router.post('/comments', async (req, res) => {
         episodeId,
         timestamp,
       });
-  
+      console.log(newComment)
```
Comment (ghost) [inline timify-api/route/comments.js]: Could we remove the console.log in code production?

File: timify-api/route/ratings.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+router.post('/ratings', async (req, res) => {
+  const { ratingValue, episodeId, userId } = req.body;
+
+    try {
+      const newRating = await Rating.create({
+        ratingValue,
+        episodeId,
+        userId,
+      });
+      console.log(newRating)
```
Comment (ghost) [inline timify-api/route/ratings.js]: Same as above

File: timify-ui/src/components/Rating/Rating.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+		try {
+			const response = await fetch("http://localhost:3000/ratings", {
+			  method: "POST",
+			  headers: { "Content-Type": "application/json" },
+			  body: JSON.stringify({ ratingValue: value, episodeId, userId }),
+			});
+	  
+			if (response.ok) {
+				setUserRated(true);
+			  console.log("Rating data posted");
```
Comment (ghost) [inline timify-ui/src/components/Rating/Rating.jsx]: Same here

File: timify-ui/src/components/Rating/Rating.jsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+	};
+
+
+	return (
+		<Container>
+			{[...Array(5)].map((item, index) => {
+				const givenRating = index + 1;
+				return (
+					<label key={index}>
+						<Radio
```
Comment (ghost) [inline timify-ui/src/components/Rating/Rating.jsx:73]: Could we format this file?

**Review by ssorenson130 — APPROVED** [review]

**Pull request merged**